packages/connect/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hoprnet/hopr-connect",
-  "version": "2.0.0-next.1",
```

Comment:
Why has this been changed to `2.0.0`? Same question applies to the other package files.

Reply:
With this PR, we are removing the current bumping version strategy.
In this step https://github.com/hoprnet/hoprnet/pull/5152/files#diff-59cda6b1dff33594abbda1fdcbde770d7a62ca1d75d63dc81d0916f7d6c124d6 We are locally modifying the package version to include the PR number, but the change is not persisted into the git repo, only locally before building the docker image.